File: core/src/com/broll/gainea/server/core/goals/impl/e1/G_MoorAndMistra.java
package com.broll.gainea.server.core.goals.impl.e1;

import com.broll.gainea.server.core.goals.OccupyGoal;
import com.broll.gainea.server.core.goals.GoalDifficulty;
import com.broll.gainea.server.core.map.impl.GaineaMap;

public class G_MoorAndMistra extends OccupyGoal {
    public G_MoorAndMistra() {
        super(GoalDifficulty.EASY, "Erobere den Kontinent Moor und die Mistra Insel");
    }

    @Override
    protected void initOccupations() {
        occupy(GaineaMap.Continents.MOOR);
        occupy(GaineaMap.Islands.MISTRAINSEL);
    }

}

File: server/src/main/com/broll/gainea/server/core/goals/impl/e1/G_MoorAndMistra.kt
package com.broll.gainea.server.core.goals.impl.e1

import com.broll.gainea.server.core.goals.GoalDifficulty
import com.broll.gainea.server.core.goals.OccupyGoal
import com.broll.gainea.server.core.map.impl.GaineaMap

class G_MoorAndMistra : OccupyGoal(GoalDifficulty.EASY, "Erobere den Kontinent Moor und die Mistra Insel") {
    override fun initOccupations() {
        occupy(GaineaMap.Continents.MOOR)
        occupy(GaineaMap.Islands.MISTRAINSEL)
    }
}
